cluster/images/kubelet/Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM centos
-ADD kubelet /kubelet
+FROM debian:jessie
```

Comment:
`kubelet` needs these shared libs at least: 
```
$ ldd kubelet
     linux-vdso.so.1 =>  (0x00007fffaa3f0000)
     libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f66f19d8000)
     libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f66f160e000)
     /lib64/ld-linux-x86-64.so.2 (0x000055a54f730000)
$ file kubelet
kubelet: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, BuildID[sha1]=83932432f65373124e2fe1c0be0b61e4c89f84e3, not stripped
```

@fgrzadkowski I guess at least some of the packages installed in the `hyperkube` `Dockerfile` needs to be installed. It might be safest to have same packages at both places.